features/product/viewing_product_in_admin_panel/viewing_product_attributes_in_different_locales.feature

```diff
@@ -0,0 +1,23 @@
+@viewing_products
```

Comment:
I'm a bit confused when it comes to articles in these scenarios - sometimes you use them and sometimes not, what can be misleading. I'd summon @CoderMaggie here :)